### PR TITLE
fix: update readme

### PR DESCRIPTION
**Pull request by clindeman**
Replace all anaconda references with miniforge. Change hand drawn red circles with straight red lines in screenshots.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,19 @@
...
+The Hazus Flood Assessment Structure Tool (FAST) calculates building-level flood impacts with user-provided building and flood depth data. FAST uses the Hazus Flood model methodology to assign depth damage functions to buildings according to their occupancy type, first floor elevation, foundation type, and number of stories. Flood depth is then extracted at every building and used as a depth damage function parameter to calculate flood losses in dollars. Flood-generated debris is estimated using building area in square feet. For more information about how FAST cacluates flood impacts, please refer to the Hazus Flood Technical Manual: https://www.fema.gov/sites/default/files/2020-09/fema_hazus_flood-model_technical-manual_2.1.pdf
```
Comment (be-cunning) [inline README.md]: Change the last sentence to the following - for an active tone & fix typo in 'calculates':

_Please refer to the Hazus Flood Technical Manual: https://www.fema.gov/sites/default/files/2020-09/fema_hazus_flood-model_technical-manual_2.1.pdf for more information about how FAST calculates flood impacts._

File: README.md [hunk 2]
```diff
@@ -1,39 +1,19 @@
...
 
 Building data must be formatted as a .csv file according to the specifications outlined here: https://github.com/nhrap-hazus/FAST/blob/master/Help/FASTBuildingData.pdf Flood depth data must be formatted as a .tiff raster. Sample building data for Honolulu, HI are included in the "UDF" folder.
 
-FAST is developed using the Hazus Python Package, HazPy. HazPy tools automatically check for updates each time they are opened. Hazus Python Package documentation is found here: https://github.com/nhrap-hazus/hazus. The Hazus Team would like to thank the Oregon Department of Geology and Mineral Industries for developing an early version of this tool: https://www.oregongeology.org/pubs/ofr/O-18-04/O-18-04_user_guide.pdf
+FAST is developed using the Hazus Python Package. This tool automatically checks for updates each time its opened. The Hazus Team would like to thank the Oregon Department of Geology and Mineral Industries for developing an early version of this tool: https://www.oregongeology.org/pubs/ofr/O-18-04/O-18-04_user_guide.pdf
```
Comment (be-cunning) [inline README.md]: Change line to _'The tool automatically checks for updates each time it runs'_

**Review by be-cunning — CHANGES_REQUESTED** [review]
Use an active tone. Adjust typo. Capitalize 'H' in GitHub.

Future reference: use 'chore' for commits like these, instead of 'fix'

File: README.md (hunk 2 above)
Comment (clindeman) [reply README.md]: Applied the change

File: README.md (hunk 1 above)
Comment (clindeman) [reply README.md]: Applied the recommendation

**Pull request merged**